specification/attestation/resource-manager/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -494,9 +494,6 @@
     "StatusResult": {
       "type": "object",
       "description": "Status of attestation service.",
-      "required": [
-        "status"
```

Comment:
Reason to remove this?

Reply:
It only occurs in response. The parameters in reponse don't need to be set as required.